Source/Glide64/Main.cpp

```diff
@@ -2038,14 +2038,14 @@ void newSwapBuffers()
       }
     }
     //hotkeys
-    if (CheckKeyPressed(G64_VK_BACK, 0x0001))
-    {
-      hotkey_info.hk_filtering = 100;
-      if (settings.filtering < 2)
-        settings.filtering++;
-      else
-        settings.filtering = 0;
-    }
+    //if (CheckKeyPressed(G64_VK_BACK, 0x0001))
+    //{
+      //hotkey_info.hk_filtering = 100;
+      //if (settings.filtering < 2)
+        //settings.filtering++;
+      //else
+        //settings.filtering = 0;
+    //}

```

Comment:
Sometimes you should try `#if 0`, because instead of deleting the 8 `//` comment-out tokens you added, you would only need to change the #if 0 to a #if 1 or whatever instead of changing 8 lines to test the code out quickly.